File: backend/insurance_engine/src/InsuranceEngine.Claims/Domain/Entities/ClaimDocument.cs
using System;

namespace InsuranceEngine.Claims.Domain.Entities;

public class ClaimDocument
{
    public Guid Id { get; set; }
    public Guid ClaimId { get; set; }
    public string DocumentType { get; set; } = string.Empty;
    public string FileUrl { get; set; } = string.Empty;
    public string FileHash { get; set; } = string.Empty;
    public bool Verified { get; set; }
    public Guid? VerifiedBy { get; set; }
    public DateTime UploadedAt { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: sdks/insuretech-go-sdk/pkg/models/claim_document.go
package models

import (
	"time"
)

// ClaimDocument represents a claim_document
type ClaimDocument struct {
	ClaimId string `json:"claim_id,omitempty"`
	CreatedAt time.Time `json:"created_at,omitempty"`
	DocumentId string `json:"document_id,omitempty"`
	DocumentType string `json:"document_type,omitempty"`
	FileHash string `json:"file_hash,omitempty"`
	FileUrl string `json:"file_url,omitempty"`
	UpdatedAt time.Time `json:"updated_at,omitempty"`
	UploadedAt time.Time `json:"uploaded_at,omitempty"`
	Verified bool `json:"verified,omitempty"`
	VerifiedBy string `json:"verified_by,omitempty"`
}
